src/main/java/cc/polyfrost/oneconfig/gui/OneConfigGui.java

```diff
@@ -273,6 +343,13 @@ public OneColor getColor() {
         return currentColorSelector.getColor();
     }
 
+    public static float getOneConfigScaleFactor() {
+        float scaleFactor = getScaleFactor();
+        if (Preferences.guiOpenAnimation)
+            return scaleFactor * INSTANCE.scaleFactor;
```

Comment:
?

Reply:
wanted to split oneconfig gui scale factor with a more general scale factor method, because other features use it too that aren't linked to oneconfig's GUI (e.g notifications)